components/progress/Circle.tsx

```diff
@@ -29,10 +29,10 @@ const Circle: React.FC<CircleProps> = props => {
     prefixCls,
     width,
     strokeWidth,
-    trailColor,
-    strokeLinecap,
+    trailColor = null as any,
```

Comment:
为啥要这样写？

```suggestion
    trailColor,
```

这样可以么

Reply:
这样的话会导致标签多生成一个属性 stroke， snap会挂掉